cve_scripts/pop_paths_comparison.py

```diff
@@ -0,0 +1,210 @@
+import os
+import json
+#add direcotry path that contains all the patch files on your local machine
```

Comment:
These comments are kinda confusing me -- is this something that the user is supposed to do or something that you are reminding yourself to do later? If it is for the user, then we definitely need a readme going here.